.github/workflows/firefox-ios-ui-tests-old.yml

```diff
@@ -0,0 +1,92 @@
+name: Focus iOS 15 & 16 tests
+on:
+  workflow_dispatch:
+
+env:
+    browser: focus-ios
+    xcode_version: 15.4
+    test_results_directory: /Users/runner/tmp
+
+jobs:
+  Focus-iOS-Tests:
+    name: Focus iOS
+    runs-on: macos-14
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - ios_version: 16.4
+            ios_simulator: iPhone 14
+          - ios_version: 15.5
+            ios_simulator: iPhone 13
+    steps:
+      - name: Check out source code
+        uses: actions/checkout@v4.1.7
+        with:
+          repository: mozilla-mobile/firefox-ios
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: '3.11'
+      - name: Install packages
+        run: |
+          brew update
+          brew install blacktop/tap/ipsw
+          pip3 install virtualenv pipenv
+      - name: Setup Xcode
+        id: xcode
+        run: |
+          sudo rm -rf /Applications/Xcode.app
+          sudo xcode-select -s /Applications/Xcode_${{ env.xcode_version }}.app/Contents/Developer
+          xcodebuild -version
+          ./checkout.sh
+          ./bootstrap.sh --force
+      - name: Create iOS ${{ matrix.ios_version }} simulator
+        id: setup-simulator
+        run: |
+          brew install blacktop/tap/ipsw
+          echo "iOS ${{ matrix.ios_version }} Simulator" | ipsw download xcode --sim
+          dmg_file=$(ls *.dmg)
+          curl https://gist.githubusercontent.com/testableapple/3688bbf7c0e475aa915bd34b7cf7876e/raw/db91bbbc5ace3835de34db5030c6b04e519847e5/install_runtime.sh > install_runtime.sh
```

Comment:
I am not happy about this solution: Some code from another repo could be gone without us knowing ahead of time. So far I have not found an official command line command to install an old runtime. We have been installing old runtime on MacStadium using Xcode IDE.

This approach works well for me so far.

Reference: https://testableapple.com/install-ios-simulator-runtimes/